bba/bba.go

```diff
@@ -511,3 +563,28 @@ func (bba *BBA) processAuxMessage(msg *pb.Message_Bba) (cleisthenes.Request, uin
 	}
 	return req, msg.Bba.Round, nil
 }
+
+func (bba *BBA) processDelayedMessage(round uint64, sender cleisthenes.Member, req cleisthenes.Request) {
+	switch r := req.(type) {
+	case *BvalRequest:
+		bba.incomingBvalReqRepo.Save(round, sender.Address, r)
+	case *AuxRequest:
+		bba.incomingAuxReqRepo.Save(round, sender.Address, r)
+	}
+}
+
+func (bba *BBA) bvalSet() bool {
+	binList := bba.binValueSet.toList()
+	if len(binList) == 0 {
+		return false
+	}
+
+	return true
+}
+
+func (bba *BBA) checkRound(round uint64) bool {
```

Comment:
how about `matchWithCurrentRound`

Reply:
isn't it too long?

Reply:
maybe but this name match better with what this function does. If there's better naming plz suggest. I comment it because `checkRound` is worst I think..